### ISSUE TITLE
Suggestion: Add asterisks to "What's happening?"

### ISSUE THREAD

**Issue by CodeGuy92** [LLK/scratch-www#5586]
![](https://u.cubeupload.com/LankyBox01/FMRleJ.png)
This user has an asterisk at the end of their name.
.
.
.
![](https://u.cubeupload.com/LankyBox01/i4CxSm.png)
But it should also show up here.

**Comment by CodeGuy92** [comment]
https://scratch.mit.edu/discuss/topic/520768/

**Comment by benjiwheeler** [comment]
I'm not positive that scratch-www gets all the info it needs to know a user is on the Scratch Team. But I _think_ it does.

**Comment by apple502j** [comment]
@benjiwheeler No, it doesn't - the API response does not include the full user object (= `scratchteam` property). Most of those other APIs, such as managers/curators API described at #5707, do, though.
